spec/controllers/home_controller_spec.rb

```diff
@@ -11,7 +11,7 @@
   let(:future) { create :far_future_rental, reservation_id: 7777 }
   let(:ongoing) { create :ongoing_rental, reservation_id: 8888 }
   let(:ongoing2) { create :ongoing_rental, reservation_id: 9999 }
-  let(:canceled) { create :mock_rental, rental_status: 'canceled', reservation_id: 0000 }
+  let(:canceled) { create :mock_rental, rental_status: 'canceled', reservation_id: 00_00 }
```

Comment:
I changed it already. It was originally like that because when the reservation_id is set the model will not create a real reservation with aggressive-epsilon.